### ISSUE TITLE
About Node Features

### ISSUE THREAD

**Issue by Peien429** [muhanzhang/pytorch_DGCNN#45]
Hi, I am a beginner of graph classification task. I have a question that if the node has a string(natural language string) attribute, how can I convert the string into node feature?
Could I use word embedding model, such as word2vec, to convert the string into node feature?
Thanks!

**Comment by muhanzhang** [comment]
Yes. You can use standard CNN/RNN over word embeddings to get sentence embeddings as node features.

**Comment by Peien429** [comment]
Thanks a lot!
